### PR TITLE
Bring in enhancements from the Kalablog8 project

### PR DESCRIPTION
**Pull request by derekderaps**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -19,9 +19,13 @@
         "composer/installers": "^1.2",
         "drupal-composer/drupal-scaffold": "^2.2",
         "cweagans/composer-patches": "~1.0",
-        "drupal/core": "~8.0",
-        "drush/drush": "~8.0",
-        "drupal/console": "~1.0"
+        "drupal/core": "~8.2",
+        "drush/drush": "8.1.3",
```
Comment (RobLoach) [inline composer.json]: Drush `8.1.3`? Not `~8.1.3`?

File: web/sites/development.services.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+# Local development services.
+parameters:
+  http.response.debug_cacheability_headers: true
+services:
+  cache.backend.null:
+    class: Drupal\Core\Cache\NullBackendFactory
```
Comment (RobLoach) [inline web/sites/development.services.yml]: Should we consider the Twig development and disabling caches too?
https://www.drupal.org/node/2598914

File: composer.json (hunk 1 above)
Comment (derekderaps) [inline composer.json]: Yeah, good question, we keep it locked at `8.1.3` due to some issues with Symfony errors in versions past this point.

File: web/sites/development.services.yml (hunk 2 above)
Comment (derekderaps) [inline web/sites/development.services.yml]: Yeah, but do you mind doing that in a separate PR? This one is just the default "development" services file provided by Drupal Scaffold.
